pyvista/plotting/volume_property.py

```diff
@@ -131,23 +171,29 @@ def apply_lookup_table(self, lookup_table: pyvista.LookupTable):
         >>> grid = pv.sample_function(
         ...     noise, [0, 3.0, -0, 1.0, 0, 1.0], dim=(40, 40, 40)
         ... )
-        >>> grid['scalars'] -= grid['scalars'].min()
-        >>> grid['scalars'] *= 255 / grid['scalars'].max()
         >>> pl = pv.Plotter()
         >>> actor = pl.add_volume(grid, show_scalar_bar=False)
-        >>> lut = pv.LookupTable(cmap='bwr')
+        >>> lut = actor.mapper.lookup_table
+        >>> lut.cmap = 'bwr'
         >>> lut.apply_opacity([1.0, 0.0, 0.0, 0.3, 0.0, 0.0, 0.0, 0.3])
-        >>> actor.prop.apply_lookup_table(lut)
         >>> pl.show()
 
         """
         if not isinstance(lookup_table, pyvista.LookupTable):
             msg = '`lookup_table` must be a `pyvista.LookupTable`'  # type: ignore[unreachable]
             raise TypeError(msg)
-
+        if self._lookup_table != lookup_table:
+            self._lookup_table = lookup_table
         self.SetColor(lookup_table.to_color_tf())
         self.SetScalarOpacity(lookup_table.to_opacity_tf())
 
+    def __del__(self):
```

Comment:
If that doesn't work, let's temporarily skip coverage.
```suggestion
    def __del__(self):  # pragma: no cover
```